``./examples/Training/python/Read_Data.py:14:1: F401 'omero' imported but unused``
I can update the doc since I have already a PR touching the file 
see https://github.com/openmicroscopy/ome-documentation/pull/1656
this will conflict otherwise

--- a/examples/Training/python/Read_Data.py
+++ b/examples/Training/python/Read_Data.py
@@ -11,7 +11,6 @@
 FOR TRAINING PURPOSES ONLY!
 """
 
-import omero
 from omero.gateway import BlitzGateway
 from Parse_OMERO_Properties import USERNAME, PASSWORD, HOST, PORT
 from Parse_OMERO_Properties import datasetId, imageId, plateId

